### PR TITLE
chore: add saifrahman

### PR DESCRIPTION
**Pull request by saifrahmn**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1053"></a>

### CODE REVIEW

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]
You can remove the slashes after the links for your blog, instagram and linkedIn has that isnt really needed and the site will load even without that.

**Comment by KendallDoesCoding** [comment]
> You can remove the slashes after the links for your blog, instagram and linkedIn has that isnt really needed and the site will load even without that.

@saifrahmn Please merge https://github.com/saifrahmn/LinkFree/pull/1 and request a re-review from me.

File: public/data/saifrahman.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
+    "name": "Saif Rahman",
+    "type": "personal",
+    "bio": "Engineer-Student-Enthusiast",
+    "avatar": "https://avatars.githubusercontent.com/u/64410006?s=400&u=bb5179ef3d88c859be1907b535cbc6b1111372e6&v=4",
```
Comment (krshdev) [inline public/data/saifrahman.json:5]:
```suggestion
    "avatar": "https://github.com/saifrahmn.png",
```

**Review by krshdev — CHANGES_REQUESTED** [review]

File: public/data/saifrahman.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        "icon": "Instagram"
+      },
+      {
+        "name": "Visit my Portfolio",
+        "url": "",
+        "icon": "Globe"
+      },
+      {
+        "name": "Contact me",
+        "url": "saifrahmn78@gmail.com",
```
Comment (krshdev) [inline public/data/saifrahman.json:34]:
```suggestion
        "url": "mailto:saifrahmn78@gmail.com",
```

File: public/data/saifrahman.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        "icon": "Hashnode"
+      },
+      {
+        "name": "Follow me on Instagram",
+        "url": "https://www.instagram.com/saif_rahm.n",
+        "icon": "Instagram"
+      },
+      {
+        "name": "Visit my Portfolio",
+        "url": "",
```
Comment (krshdev) [inline public/data/saifrahman.json:29]: You can remove the portfolio section if you don't have one 😅

**Review by emmalearnscode — APPROVED** [review]

File: public/data/saifrahman.json (hunk 1 above)
Comment (KendallDoesCoding) [reply public/data/saifrahman.json]: @saifrahmn Please mark this change as resolved.

File: public/data/saifrahman.json (hunk 3 above)
Comment (KendallDoesCoding) [reply public/data/saifrahman.json]: @saifrahmn Please mark this change as resolved.

File: public/data/saifrahman.json (hunk 2 above)
Comment (KendallDoesCoding) [reply public/data/saifrahman.json]: @saifrahmn Please mark this change as resolved.

File: public/data/saifrahman.json [hunk 4]
```diff
@@ -0,0 +1,40 @@
+{
+    "name": "Saif Rahman",
+    "type": "personal",
+    "bio": "Engineer-Student-Enthusiast",
```
Comment (syedsohan) [inline public/data/saifrahman.json:4]:
```suggestion
    "bio": "Engineer | Student | Enthusiast",
```

File: public/data/saifrahman.json [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+        "icon": "Instagram"
+      },
+      
+      {
+        "name": "Contact me",
+        "url": "mailto:saifrahmn78@gmail.com",
+        "icon": "Envelope"
+      },
+      {
+      "name": "Follow me on linkedIn",
```
Comment (syedsohan) [inline public/data/saifrahman.json:34]:
```suggestion
      "name": "Follow me on LinkedIn",
```

**Review by syedsohan — CHANGES_REQUESTED** [review]

**Review by KendallDoesCoding — APPROVED** [review]

**Review by syedsohan — APPROVED** [review]

File: public/data/saifrahman.json [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+        "url": "mailto:saifrahmn78@gmail.com",
+        "icon": "Envelope"
+      },
+      {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/saif-rahman-a62686201",
+      "icon": "LinkedIn"
+    }
+    ]
+
```
Comment (Panquesito7) [inline public/data/saifrahman.json:39]:
```suggestion

```

Comment (krshdev) [reply public/data/saifrahman.json:39]: Formatting will be done by gh-action? 😅

**Review by krshdev — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
LGTM! Thanks for the contribution :)

**Review by schmelto — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Review by ssensalo — APPROVED** [review]

**Pull request merged**